### PR TITLE
changing background to off white

### PR DESCRIPTION
**Pull request by laurafitzgerald**
# Description

Changing the background of the main container to off white

## What

See above

## Why

The white background appears very bright now after adding the background image

## How

By updating the css that is used to style the background

## Verification Steps

Check out the branch, view the color. voila!

# Checklist:

- [ ] This pr is linked to the related trello card
- [ ] My code follows the style guidelines of this project
- [ ] I have requested a code review from (at least) two team members
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] This change has been verified by another team member

### CODE REVIEW

**Comment by ciaranRoche** [comment]
:eyes:

**Comment by ciaranRoche** [comment]
+1 makes a huge difference, and I am all in favor of this change.

**Comment by laurafitzgerald** [comment]
@ciaranRoche updated the logo image as well. would you mind verifying this works.

File: assets/css/style.css [hunk 1]
```diff
@@ -6,6 +6,6 @@ body {
 }
 
 div.ui.divbg.container {
-    background-color:white;
+    background-color:#fbfbfb;
```
Comment (laurafitzgerald) [inline assets/css/style.css]: @ciaranRoche can we remove this now?

**Comment by laurafitzgerald** [comment]
@dimitraz I've rebased this now. Can you give it a quick look to make sure it's correct. Had to resolve conflict in the css style sheet

**Pull request merged**